app/controllers/help_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class HelpController < ApplicationController
+  def show; end
```

Comment:
Do you need empty controller actions for pages in rails 5+?

Reply:
Doesn't look like they're needed.
http://guides.rubyonrails.org/layouts_and_rendering.html#rendering-by-default-convention-over-configuration-in-action